pkg/connector/tickets.go

```diff
@@ -45,11 +45,10 @@ func getJiraStatusesForProject(ctx context.Context, client *jira.Client, project
 
 		jiraStatuses = append(jiraStatuses, statuses...)
 
-		totalGot := statusOffset + resp.MaxResults
-		if totalGot >= resp.Total {
+		if statusOffset >= resp.Total {
 			break
 		}
-		statusOffset += resp.Total
+		statusOffset += resp.MaxResults
```

Comment:
`statusOffset += resp.MaxResults` should happen before the `statusOffset >= resp.Total` check

Reply:
https://github.com/ConductorOne/baton-jira/pull/13/commits/a80ecab3a2301b370b9f7eb56957e4aed1ee2f0c